### DISCUSSION TITLE
Can user create a fresh account with the same social login

### DISCUSSION THREAD

**Discussion by skambalin** [Web3Auth/Web3Auth#1193] [Help]
Example:
- User lost his device and seed phrase so he is not able to pass 2fa screen and access his account
- He does not care about his old account
- He wants to create a fresh account with the same social login, but different seed phrase  

How can he do it? I could not find a solution myself, please advice if it is possible or not.

**Comment by shahbaz17** [comment]
Hey @skambalin 

It is not possible to create a new account with the same social login.
